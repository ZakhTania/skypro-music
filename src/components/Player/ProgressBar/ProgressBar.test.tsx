import renderer from "react-test-renderer";
import { ProgressBar } from ".";
describe("Компонент полоса прогресса", () => {
  it("корректно отрисовывает разметку с начальными значениями", () => {
    const mockSetCurrentProgress = jest.fn();
    const mockHandleProgressChange = jest.fn();
    const component = renderer
      .create(
        <ProgressBar
          currentProgress={0}
          setCurrentProgress={mockSetCurrentProgress}
          duration={100}
          handelProgressChange={mockHandleProgressChange}
        />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
