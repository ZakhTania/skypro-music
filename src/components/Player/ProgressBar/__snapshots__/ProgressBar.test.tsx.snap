// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент полоса прогресса корректно отрисовывает разметку с начальными значениями 1`] = `
<input
  className="styledProgressInput"
  max={100}
  min={0}
  onChange={[Function]}
  step={0.01}
  type="range"
  value={0}
/>
`;
